scripts/check-license-headers.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+lic_hdr_txt=`cat license-header.txt`
+lic_hdr_line_count=`echo ${lic_hdr_txt} | wc -l`
+lic_hdr_target_txt="Copyright 2016 Capital One Services, LLC"
+
+read -d '' spdx_lines <<EOF
```

Comment:
would this special spdx check still be needed since you've already resolved the license for the entire project?